example/lib/views/environment_dialog.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
```

Comment:
For this file, we plan to remove the development environment on the public repository. Let's take this in further discussion.

Reply:
@chiragphyllo what I need to do ? let me know , what is the further need to be done here ?

Reply:
For this Pankaj, we need to remove the development environment from the public repository.
Internally, we can have this development environment for testing, but for public we need to remove this as this might can confuse the developers.

Reply:
@chiragphyllo please make one private repo for sample app so I can upload for internal QA testing .

Reply:
@erpankajpatel , we can do something like this - https://stackoverflow.com/questions/7983204/having-a-private-branch-of-a-public-repo-on-github